### DISCUSSION TITLE
Copying tags from Lambdas to their default created Cloudwatch Log groups

### DISCUSSION THREAD

**Discussion by IshaqIdris** [cloud-custodian/cloud-custodian#8525] [AWS]
Hi everyone!

I'm trying to copy tags down from a lambda to their automatically created CloudWatch Log groups, this helps us put ownership to the Log Groups as well. When using the `copy-related-tags` action is it possible to pass in a regex value for the `key`?

I was thinking I'd use the CloudWatch Log Group attribute `LogGroupName` with a regex like `\/aws\/lambda\/(,?.*)` as the `key` because all lambda default logs start with that prefix `/aws/lambda/<lambda function name>`. Therefore it should in theory pic up the tags from a relevant lambda?

Has anyone been able to get something like this working, or done the copying of lambda tags down to Cloud Watch logs differently?

Any help appreciated!

Cheers,
Shaq

**Comment by free88dom** [comment]
Probably late to the party, but i'm dealing with something similar right now, and for such created log-group by Lambda, i'm using 
```   
 actions:
      - type: auto-tag-user
         tag: "LambdaFunctionName"
```
it's triggered by `cloudtrail` mode looking for `CreateLogGroup` triggered by Lambda function
